wordpress-mobile/WordPress-Aztec-iOS
# Support alternate bullet style per level

Similar to web browsers, draw different bullets at different levels.

To test:
- Create an unordered list
- Add multiple levels to it
- Notice the bullet style for level one is different than level two and three

| Before | After |
| ------ | ------ |
| ![old](https://user-images.githubusercontent.com/10981/160511981-8ae4d3f2-537e-4d18-a705-5641d59f255d.png) | ![new](https://user-images.githubusercontent.com/10981/160512005-04669be2-ea56-44df-a3af-2fdbb643027c.png) |
